### DISCUSSION TITLE
VLANS on Lan

### DISCUSSION THREAD

**Discussion by molesza** [Ysurac/openmptcprouter#1946] [General]
Hi. I would like to setup multiple VLANS on the lan side. How do I add Vlans for LAN? I have a guest network on my previous openwrt router and I would like to do the same in OMR.

Thanks

**Comment by bjoerrrrn** [comment]
You can add the interface with a . and then the VLAN-ID. Here it is eth0.104 for example on eth0 interface VLAN 104.
